unizip-readme.txt

```diff
@@ -4,25 +4,23 @@ To run the game, extract the engine archive for your operating system
 into this subdirectory, so the files are right outside the pkg/ directory.
 Then, run the executable extracted from the archive.
 
-Engine archives are named like <system>-<architecture>.zip
-
-System is either, linux, windows, or macos.
-
-Architecture is either:
-
-- amd64: 64-bit x86 processor
-- i686: 32-bit x86 processor
-- arm64: 64-bit arm processor
-- armhf: 32-bit arm processor with hardware float
-
-For example windows-amd64.zip is the engine archive for running
-the game on the Windows operating system for 64-bit x86 computers.
+NOTE: In Windows Explorer and many Linux file browsers, using the
+"Extract all" function creates an unwanted extra directory, so it is better
+to put the archive's files into this directory using copy and paste.
+
+Archive for each operating system:
+	Linux, x86_64 (most common): linux-amd64.zip
+	Linux, 32-bit x86: linux-i686.zip
+	Linux, 64-bit ARM (AArch64): linux-arm64.zip
+	Linux, 32-bit ARM: linux-armhf.zip
+	Windows, 64-bit: windows-amd64.zip
+	Windows, 32-bit: windows-i686.zip
+	MacOS (either Intel or Apple Silicon): macos-amd64.zip
```

Comment:
Why not more information:

```
Archive for each operating system:

Linux, 64-bit x86 (also known as x86_64 or x86-64), most common:
	linux-amd64.zip
Linux, 32-bit x86, rare, very old computers:
	linux-i686.zip
Linux, 64-bit ARM (also known as AArch64):
	linux-arm64.zip
Linux, 32-bit ARM (not armel, requires hardware float):
	linux-armhf.zip
Windows, 64-bit (also known as x64), Windows 11, common with Windows 7, 8 or 10:
	windows-amd64.zip
Windows, 32-bit, common with low-end computers running Windows 7, 8 or 10:
	windows-i686.zip
MacOS, either Intel or Apple Silicon:
	macos-amd64.zip
```

For example Ubuntu writes “_Ubuntu 22.04.3 Desktop (64-bit)_” in [download page](https://ubuntu.com/download/alternative-downloads), so we better start with that `64-bit` token too.

Actually the standard name of `x86_64` is in fact `x86-64` (see [x86-64](https://en.wikipedia.org/wiki/X86-64)). We probably not have to mention `EM64T` since it's much more confidential.

The Linux `armel` variant for 32-bit arm was very common in the past before almost all distros migrated to armhf as “Modern 32-bit ARM” (Debian started armhf with Wheezy for example), armel distros are still distributed and compatible with latest armhf devices. Android still uses armel for their 32-bit build, even on armhf devices, so this name is still widely used and in mind of many people and may be believed to be the architecture variant to pick. So we better make it obvious armhf is not armel to remove any confusion. This will help those wanting to run a server on ARM to know what Linux distro flavour to install by reading this readme.

Some `32-bit x86 Linux` installs may  be found with some virtual machines, but I assume those users would know what to use, so I only mention “_very old computers_”.

Reply:
Updated